Q24ECU/core/include/interfaces/interface_uart.h

```diff
@@ -13,6 +13,10 @@
 #include "semphr.h"
 #include "stream_buffer.h"
 
+#define UART_WRITE_OK 0
+#define UART_ERR_UNDEF 1
+#define UART_ERR_ACC 2
```

Comment:
Nit: Could be replaced with an enum - not mandatory, more of a style preference.